doc/configcommands.dsv

```diff
@@ -2,7 +2,7 @@ always-display-description||[yes/no]||no||If set to `yes`, then the description
 always-download||<url> [<url>...]||n/a||Specifies one or more feed URLs that should always be downloaded, regardless of their Last-Modified timestamp and ETag header. This option can be specified multiple times.||always-download "https://www.n-tv.de/23.rss"
 article-sort-order||<sortfield>[-<direction>]||date||The <sortfield> specifies which article property shall be used for sorting, currently available are: `date`, `title`, `flags`, `author`, `link`, `guid` and `random`. The optional <direction> specifies the sort direction. `asc` specifies ascending sorting, `desc` specifies descending sorting. Note that direction does not affect `random` sort order. For `date`, `desc` is default, for all others, `asc` is default.||article-sort-order author-desc
 articlelist-format||<format>||"%4i %f %D %6L  %?T?|%-17T|  ?%t"||This variable defines the format of entries in the article list. See the respective section in the documentation for more information on format strings.||articlelist-format "%4i %f %D   %?T?|%-17T|  ?%t"
-articlelist-title-format||<format>||"%N %V - Articles in feed '%T' (%u unread, %t total)%?F? matching filter `%F'&? - %U"||Format of the title in article list. See "Format Strings" section of Newsboat manual for details on available formats.||articlelist-title-format "Articles in feed '%T' (%u unread)"
+articlelist-title-format||<format>||"%N %V - Articles in feed '%T' (%u unread, %t total)%?F? matching filter `%F'&? - %U (localized)"||Format of the title in article list. See "Format Strings" section of Newsboat manual for details on available formats.||articlelist-title-format "Articles in feed '%T' (%u unread)"
```

Comment:
I think the "(localized)" note should actually be placed outside of double quotes, otherwise it looks like it's a part of the default value. Can you please fix this throughout the PR?
```suggestion
articlelist-title-format||<format>||"%N %V - Articles in feed '%T' (%u unread, %t total)%?F? matching filter `%F'&? - %U" (localized)||Format of the title in article list. See "Format Strings" section of Newsboat manual for details on available formats.||articlelist-title-format "Articles in feed '%T' (%u unread)"
```

Reply:
Fixed @ e153ee3df65d7cd638f41e4881d70cdcbccb7b17